src/components/Translation.tsx

```diff
@@ -20,14 +19,41 @@ const transform = {
 // Renders the translation string for the given translation key `id`. It
 // fallback to English if it doesn't find the given key in the current language
 const Translation = ({ id, options }: Props) => {
-  // const { t } = useTranslation()
+  const [requiredNamespaces, setRequiredNamespaces] = useState<string[]>([])
```

Comment:
Side question... I recall at one point when we first started using TypeScript we tried to use `Array<T>` syntax instead of `T[]`... I'm fairly indifferent on this, but just noticed this and figured I'd check if we have a preferred approach. cc: @pettinarip 

Maybe another small thing to add to the docs

Reply:
From what I've seen since we added TS, most devs seems to prefer the `[]` syntax, so I think we should go in that direction.

I agree with you, this could be something we add to the conventions.

Reply:
Okay, yeah I agree... feel like we were swimming upstream a little using `Array<>`